app/services/recommend_service.py

```
@@ -0,0 +1,102 @@
+import numpy as np
+import pandas as pd
+from scipy.sparse import coo_matrix
+from implicit.als import AlternatingLeastSquares
+from sqlalchemy.orm import Session
+from app.models import BrandClickLog, StoreClickLog, BrandEmbedding, Store, Brand
```

Ruff (0.12.2)

`app.models.Brand` imported but unused

Remove unused import: `app.models.Brand`

(F401)